app/javascript/packs/application.js

```diff
@@ -7,9 +7,13 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
-require("jquery")
-import "bootstrap"
-
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+
+require("jquery")
+import "bootstrap"
+
+//= require i18n
+//= require i18n.js
+//= require i18n/translations
```

Comment:
thay toàn bộ code file này thành 
import Rails from "@rails/ujs"
import Turbolinks from "turbolinks"
import * as ActiveStorage from "@rails/activestorage"
import "channels"
import "bootstrap"
import I18n from "i18n-js"
require("jquery")

window.I18n = I18n
Rails.start()
Turbolinks.start()
ActiveStorage.start()